Task-27/index.js

```diff
@@ -0,0 +1,24 @@
+const input = document.createElement('input')
+const button = document.createElement('button')
+const body = document.querySelector('body')
+
+button.innerText = 'GO'
+
+body.append(input, button)
+
+body.style.display = 'flex';
+body.style.justifyContent = 'center';
+body.style.paddingTop = '50vh';
+
+button.style.width = '50px'
+button.style.backgroundColor = 'grey'
+button.style.marginLeft = '10px'
+
+button.addEventListener('click', (ev) => {
+    if (input.value.startsWith('https')|input.value.startsWith('http')){
```

Comment:
```suggestion
    if (input.value.startsWith('https') || input.value.startsWith('http')){
```

` | ` оператор побітового або, Вам пощастило ще це спрацювало в цьому випадку, але будьте уважні і більше так не робіть

` || ` оператор логічного або